selfdrive/car/mazda/carstate.py

```diff
@@ -18,118 +18,169 @@ def __init__(self, CP):
     self.lkas_allowed_speed = False
     self.lkas_disabled = False
 
-  def update(self, cp, cp_cam):
+  def update(self, cp, cp_cam, cp_body):
 
     ret = car.CarState.new_message()
-    ret.wheelSpeeds = self.get_wheel_speeds(
```

Comment:
Let's try to make the signal names match so we don't need to split this, but if we can't let's split the update function into different gens like Hyundai CAN-FD does.